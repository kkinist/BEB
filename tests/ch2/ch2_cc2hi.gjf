%chk=ch2.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check

BEB step 9: high-spin CCSD(T) for dication of ch2

2 3

C H 0
cc-pVTZ
****

