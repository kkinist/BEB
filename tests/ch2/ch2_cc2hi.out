 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-27170.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     27172.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch2.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check
 ---------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,8=3,13=1,38=6/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -------------------------------------------------
 BEB step 9: high-spin CCSD(T) for dication of ch2
 -------------------------------------------------
 Structure from the checkpoint file:  "ch2.chk"
 Charge =  2 Multiplicity = 3
 Redundant internal coordinates found in file.
 C,0,0.,0.,0.0661340759
 H,0,0.,-0.9932409277,-0.3639822479
 H,0,0.,0.9932409277,-0.3639822479
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.066134
      2          1           0        0.000000   -0.993241   -0.363982
      3          1           0        0.000000    0.993241   -0.363982
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  C    0.000000
     2  H    1.082371   0.000000
     3  H    1.082371   1.986482   0.000000
 Stoichiometry    CH2(2+,3)
 Framework group  C2V[C2(C),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.107529
      2          1           0        0.000000    0.993241   -0.322587
      3          1           0        0.000000   -0.993241   -0.322587
 ---------------------------------------------------------------------
 Rotational constants (GHZ):   1582.9318105    254.1515972    218.9909540
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    27 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     7 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    12 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    19 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    23 symmetry adapted basis functions of A1  symmetry.
 There are     7 symmetry adapted basis functions of A2  symmetry.
 There are    11 symmetry adapted basis functions of B1  symmetry.
 There are    17 symmetry adapted basis functions of B2  symmetry.
    58 basis functions,    86 primitive gaussians,    65 cartesian basis functions
     4 alpha electrons        2 beta electrons
       nuclear repulsion energy         6.1332544348 Hartrees.
 NAtoms=    3 NActive=    3 NUniq=    2 SFac= 2.25D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    58 RedAO= T EigKep=  2.61D-03  NBF=    23     7    11    17
 NBsUse=    58 1.00D-06 EigRej= -1.00D+00 NBFU=    23     7    11    17
 Initial guess from the checkpoint file:  "ch2.chk"
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (A1) (B2) (A1) (B1) (A1) (B2) (A2)
                 (A1) (B1) (B2) (A1) (B1) (B2) (A2) (A1) (B2) (A1)
                 (A1) (B1) (B2) (A1) (B2) (B1) (A2) (A1) (B2) (B1)
                 (A1) (A2) (A1) (B1) (B2) (A1) (B2) (A2) (B1) (A1)
                 (B2) (B2) (A2) (A1) (B1) (A1) (B2) (A2) (B1) (A1)
                 (A1) (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1)
       Virtual   (B2) (B1) (A1) (A1) (B2) (A1) (B2) (B1) (A1) (A1)
                 (B2) (A2) (A1) (B1) (B2) (A1) (B1) (B2) (A2) (A1)
                 (B2) (A1) (B1) (A1) (B2) (A1) (B2) (A2) (B1) (A1)
                 (B2) (B1) (A1) (A2) (A1) (B1) (B2) (A1) (B2) (A2)
                 (B1) (B2) (A1) (B2) (A2) (A1) (B1) (A1) (B2) (A2)
                 (B1) (A1) (A1) (B2) (B2) (A1)
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0039 S= 1.0013
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=3449881.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -37.6313414025     A.U. after   10 cycles
            NFock= 10  Conv=0.47D-08     -V/T= 1.9861
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0081 S= 1.0027
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     2.0081,   after     2.0000
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    58
 NBasis=    58 NAE=     4 NBE=     2 NFC=     1 NFV=     0
 NROrb=     57 NOA=     3 NOB=     1 NVA=    54 NVB=    56

 **** Warning!!: The largest alpha MO coefficient is  0.10275998D+02


 **** Warning!!: The largest beta MO coefficient is  0.10271153D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             3 LenV=    1048291445
 LASXX=        68438 LTotXX=       68438 LenRXX=      141786
 LTotAB=       73348 MaxLAS=      260091 LenRXY=           0
 NonZer=      210224 LenScr=      785920 LnRSAI=      260091
 LnScr1=      785920 LExtra=           0 Total=      1973717
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             1 LenV=    1048291445
 LASXX=        23328 LTotXX=       23328 LenRXX=       45621
 LTotAB=       22293 MaxLAS=       86697 LenRXY=           0
 NonZer=       68949 LenScr=      785920 LnRSAI=       86697
 LnScr1=      785920 LExtra=           0 Total=      1704158
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   1.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.2605619844D-02 E2=     -0.9918721977D-02
     alpha-beta  T2 =       0.1688377956D-01 E2=     -0.5259398001D-01
     beta-beta   T2 =       0.0000000000D+00 E2=      0.0000000000D+00
 ANorm=    0.1009697677D+01
 E2 =    -0.6251270199D-01 EUMP2 =    -0.37693854104511D+02
 (S**2,0)=  0.20081D+01           (S**2,1)=  0.20010D+01
 E(PUHF)=      -0.37633525020D+02        E(PMP2)=      -0.37695075139D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=3411002.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 E(PMP3)=      -0.37706874463D+02
 MP4(R+Q)=  0.11063353D-02
 E3=       -0.12444490D-01        EUMP3=      -0.37706298595D+02
 E4(DQ)=   -0.27397702D-02        UMP4(DQ)=   -0.37709038365D+02
 E4(SDQ)=  -0.30405044D-02        UMP4(SDQ)=  -0.37709339099D+02
 DE(Corr)= -0.73855959E-01 E(Corr)=     -37.705197361    
 NORM(A)=   0.10161883D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.77937465E-01 E(CORR)=     -37.709278868     Delta=-4.08D-03
 NORM(A)=   0.10186930D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79243306E-01 E(CORR)=     -37.710584709     Delta=-1.31D-03
 NORM(A)=   0.10193243D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79521933E-01 E(CORR)=     -37.710863336     Delta=-2.79D-04
 NORM(A)=   0.10194495D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79553654E-01 E(CORR)=     -37.710895056     Delta=-3.17D-05
 NORM(A)=   0.10194629D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79555648E-01 E(CORR)=     -37.710897050     Delta=-1.99D-06
 NORM(A)=   0.10194673D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79556962E-01 E(CORR)=     -37.710898365     Delta=-1.31D-06
 NORM(A)=   0.10194677D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79557167E-01 E(CORR)=     -37.710898570     Delta=-2.05D-07
 NORM(A)=   0.10194677D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        10
 NAB=     3 NAA=     3 NBB=     0.
 DE(Corr)= -0.79557078E-01 E(CORR)=     -37.710898481     Delta= 8.86D-08
 NORM(A)=   0.10194674D+01
 Largest amplitude= 7.74D-02
 Time for triples=        0.65 seconds.
 T4(CCSD)= -0.14735379D-02
 T5(CCSD)=  0.19934349D-04
 CCSD(T)= -0.37712352085D+02

 S**2, projected HF & approx projected MPn energies after annihilation of
 unwanted spin states (see manual for definitions):

 spins       (S**2,0) (S**2,1)    PUHF        PMP2        PMP3        PMP4
 annihilated
 s+1         2.00000  2.00000  -37.633525  -37.695075  -37.706874

 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (A1) (B2) (A1) (B1) (A2) (B2) (A1)
                 (A1) (B1) (B2) (A1) (B1) (B2) (A2) (A1) (B2) (A1)
                 (A1) (B1) (B2) (A1) (B2) (A2) (B1) (A1) (B2) (B1)
                 (A1) (A2) (A1) (B1) (B2) (A1) (B2) (A2) (B1) (A1)
                 (B2) (B2) (A1) (A2) (B1) (A1) (B2) (A2) (B1) (A1)
                 (A1) (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1)
       Virtual   (B2) (B1) (A1) (A1) (B2) (A1) (B2) (B1) (A1) (A1)
                 (A2) (B2) (A1) (B1) (B2) (A1) (B1) (B2) (A2) (A1)
                 (B2) (A1) (B1) (A1) (B2) (A1) (B2) (A2) (B1) (A1)
                 (B2) (B1) (A1) (A2) (A1) (B1) (B2) (A1) (B2) (A2)
                 (B1) (B2) (A1) (B2) (A2) (A1) (B1) (A1) (B2) (A2)
                 (B1) (A1) (A1) (B2) (B2) (A1)
 The electronic state is 3-B2.
 Alpha  occ. eigenvalues --  -12.26919  -1.81712  -1.56510  -1.32447
 Alpha virt. eigenvalues --   -0.72829  -0.41708  -0.35088  -0.25060  -0.18923
 Alpha virt. eigenvalues --   -0.17109  -0.14091  -0.00880   0.00216   0.01155
 Alpha virt. eigenvalues --    0.09572   0.10802   0.28999   0.37419   0.46157
 Alpha virt. eigenvalues --    0.54580   0.63529   0.71117   0.80592   1.05635
 Alpha virt. eigenvalues --    1.49607   1.50944   1.55482   1.61711   1.95906
 Alpha virt. eigenvalues --    1.99561   2.04982   2.07408   2.18634   2.21201
 Alpha virt. eigenvalues --    2.25281   2.36263   2.37185   2.42730   2.50458
 Alpha virt. eigenvalues --    2.72620   2.78204   2.95707   3.07754   3.14696
 Alpha virt. eigenvalues --    3.15174   3.34575   3.37470   3.37561   3.45502
 Alpha virt. eigenvalues --    4.01174   4.25948   4.31550   4.38790   4.47623
 Alpha virt. eigenvalues --    4.65645   4.94655   5.12692  10.49005
  Beta  occ. eigenvalues --  -12.22281  -1.56249
  Beta virt. eigenvalues --   -0.91572  -0.67842  -0.63090  -0.37184  -0.27672
  Beta virt. eigenvalues --   -0.19391  -0.13597  -0.12579  -0.08698   0.03677
  Beta virt. eigenvalues --    0.04317   0.07961   0.15959   0.16127   0.33009
  Beta virt. eigenvalues --    0.41316   0.48190   0.60012   0.65733   0.75419
  Beta virt. eigenvalues --    0.85177   1.09246   1.53229   1.60262   1.61655
  Beta virt. eigenvalues --    1.70858   2.01326   2.05623   2.09848   2.11661
  Beta virt. eigenvalues --    2.27015   2.27419   2.36175   2.41959   2.43786
  Beta virt. eigenvalues --    2.49388   2.60235   2.76745   2.83469   2.99615
  Beta virt. eigenvalues --    3.10508   3.17790   3.17811   3.37295   3.40390
  Beta virt. eigenvalues --    3.41833   3.47735   4.04490   4.29445   4.34212
  Beta virt. eigenvalues --    4.42038   4.53353   4.69607   4.99974   5.16883
  Beta virt. eigenvalues --   10.51740
          Condensed to atoms (all electrons):
               1          2          3
     1  C    4.588870   0.262677   0.262677
     2  H    0.262677   0.177355   0.002856
     3  H    0.262677   0.002856   0.177355
          Atomic-Atomic Spin Densities.
               1          2          3
     1  C    1.345017   0.111791   0.111791
     2  H    0.111791   0.105378  -0.001469
     3  H    0.111791  -0.001469   0.105378
 Mulliken charges and spin densities:
               1          2
     1  C    0.885776   1.568599
     2  H    0.557112   0.215700
     3  H    0.557112   0.215700
 Sum of Mulliken charges =   2.00000   2.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  C    2.000000   2.000000
 Electronic spatial extent (au):  <R**2>=             11.7013
 Charge=              2.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=             -0.8936  Tot=              0.8936
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -3.1477   YY=              1.9271   ZZ=             -3.7082
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -1.5048   YY=              3.5701   ZZ=             -2.0653
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             -0.7320  XYY=              0.0000
  XXY=              0.0000  XXZ=             -0.0502  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             -2.2069  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -1.8526 YYYY=              1.5371 ZZZZ=             -3.7878 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -1.2205 XXZZ=             -0.9381 YYZZ=             -0.8089
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 6.133254434788D+00 E-N=-9.492979291835D+01  KE= 3.816179947613D+01
 Symmetry A1   KE= 3.687588005783D+01
 Symmetry A2   KE= 1.847186108663D-34
 Symmetry B1   KE= 6.170141917005D-33
 Symmetry B2   KE= 1.285919418301D+00
 1\1\GINC-GAMBA\SP\UCCSD(T)-FC\Gen\C1H2(2+,3)\IRIKURA\17-May-2017\0\\# 
 CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check\\BEB ste
 p 9: high-spin CCSD(T) for dication of ch2\\2,3\C,0,0.,0.,0.0661340759
 \H,0,0.,-0.9932409277,-0.3639822479\H,0,0.,0.9932409277,-0.3639822479\
 \Version=ES64L-G09RevD.01\State=3-B2\HF=-37.6313414\MP2=-37.6938541\MP
 3=-37.7062986\MP4D=-37.7101447\MP4DQ=-37.7090384\PUHF=-37.633525\PMP2-
 0=-37.6950751\PMP3-0=-37.7068745\MP4SDQ=-37.7093391\CCSD=-37.7108985\C
 CSD(T)=-37.7123521\S2=2.008055\S2-1=2.000953\S2A=2.\RMSD=4.668e-09\PG=
 C02V [C2(C1),SGV(H2)]\\@


 ALL THINGS ARE POSSIBLE EXCEPT SKIING THROUGH A REVOLVING DOOR.
 Job cpu time:       0 days  0 hours  0 minutes 10.2 seconds.
 File lengths (MBytes):  RWF=     22 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 12:52:49 2017.
